public/app/features/explore/ExploreToolbar.tsx

```diff
@@ -149,25 +150,34 @@ export function ExploreToolbar({ exploreId, topOfViewRef, onChangeTime }: Props)
             <ToolbarButton
               variant="canvas"
               key="split"
-              tooltip="Split the pane"
+              tooltip={t('explore.toolbar.split.tooltip', 'Split the pane')}
               onClick={onOpenSplitView}
               icon="columns"
               disabled={isLive}
             >
-              Split
+              <Trans i18nKey="explore.toolbar.split.title"> Split </Trans>
             </ToolbarButton>
           ) : (
             <ButtonGroup key="split-controls">
               <ToolbarButton
                 variant="canvas"
-                tooltip={`${isLargerPane ? 'Narrow' : 'Widen'} pane`}
+                tooltip={
+                  isLargerPane
+                    ? t('explore.toolbar.split.narrow', 'Narrow pane')
+                    : t('explore.toolbar.split.widen', 'Widen pane')
```

Comment:
Maybe `explore.toolbar.split-widen`?

Reply:
Sure, thanks for your feedback. addressed it here.
```suggestion
                    : t('explore.toolbar.split-widen', 'Widen pane')
```